### DISCUSSION TITLE
How do I orderby latest on a table

### DISCUSSION THREAD

**Discussion by ppshobi** [filamentphp/filament#1795] [Help]
I have a table of products 
```
 return $table
            ->columns([
                Tables\Columns\TextColumn::make('name')->wrap(),
               ...
               ....
            ])
```

Currently, the first created products are shown in the initial rendering of the table, I want to show the latest products first. I don't want to add a sortable column  as in this one https://github.com/laravel-filament/filament/discussions/261

**Comment by danharrin** [comment]
#1802. In the next release, you can specify `->defaultSort('created_at', 'desc')` and it will work without a column present.
